source/common/http/conn_manager_impl.cc

```diff
@@ -756,6 +767,25 @@ void ConnectionManagerImpl::ActiveStream::resetIdleTimer() {
 
 void ConnectionManagerImpl::ActiveStream::onIdleTimeout() {
   connection_manager_.stats_.named_.downstream_rq_idle_timeout_.inc();
+
+  if (Runtime::runtimeFeatureEnabled(
```

Comment:
Can you add a comment above this line that says something like: "See below for more information on this block."

Reply:
done